dotexporter.py

```diff
@@ -124,11 +226,21 @@ def do_GET(self):
           'name': 'dot_rpc_healthy',
           'value': 1
         })
+        m.append({
+          'name': 'dot_latest_best',
+          'value': self.previous['best_epoch']
+        })
+        m.append({
+          'name': 'dot_latest_finalized',
+          'value': self.previous['finalized_epoch']
+        })
```

Comment:
I would suggest to also do it the way as it is now for the `dot_chain_stall_drift` metric e.g. metric name `dot_chain_latest` with property either `best` or `finalized`.